pkgs/applications/graphics/rawtherapee/default.nix

```diff
@@ -1,29 +1,34 @@
-{ stdenv, fetchurl, pkgconfig, gtk, cmake, pixman, libpthreadstubs, gtkmm, libXau
-, libXdmcp, lcms2, libiptcdata, libcanberra, fftw, expat
+{ stdenv, fetchFromGitHub, pkgconfig, gtk, cmake, pixman, libpthreadstubs, gtkmm, libXau
+, libXdmcp, lcms2, libiptcdata, libcanberra, fftw, expat, pcre, libsigcxx 
 , mercurial  # Not really needed for anything, but it fails if it does not find 'hg'
 }:
 
 stdenv.mkDerivation rec {
-  name = "rawtherapee-4.0.10";
+  name = "rawtherapee-4.2";
```

Comment:
I suggest factoring out a `version` attribute and re-using it in `name` and `rev` below to make updating easier.